doc/user/content/get-started.md

```diff
@@ -147,15 +158,15 @@ For this guide, you'll use a [built-in load generator](/sql/create-source/load-g
     <p></p>
 
     ```nofmt
-              name          |      type      | size
-    ------------------------+----------------+------
-     accounts               | subsource      |
-     auction_house          | load-generator |
-     auction_house_progress | subsource      |
-     auctions               | subsource      |
-     bids                   | subsource      |
-     organizations          | subsource      |
-     users                  | subsource      |
+                name            |      type      | size
+    ----------------------------+----------------+------
+     accounts                   | subsource      |
+     auction_house_qck          | load-generator |
```

Comment:
Thanks for catching that source name confusion. I had experimented with changing the object names before deciding to reintroduce a schema, and I missed cleaning up this bit. 

Removed `TICK INTERVAL` as well. I added that because I wanted to make sure there's enough data to show meaningful latency differences pre- and post-index. But it's probably unnecessary.